src/styles/_rezume.scss

```diff
@@ -1,7 +1,7 @@
 .rezume-rezume {
   @include box-shadow(5px);
   background-color: #fff;
-  padding: 75px 100px;
+  padding: 75px 75px;
```

Comment:
If it's both the same, just remove the second value.

Reply:
Made the changes